packages/matter-node-ble.js/src/ble/BleBroadcaster.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BlenoBleServer } from "./BlenoBleServer";
+import { VendorId } from "@project-chip/matter.js/datatype";
+import {
+    InstanceBroadcaster, CommissionerInstanceData, CommissioningModeInstanceData
+} from "@project-chip/matter.js/common";
+import { Logger } from "@project-chip/matter.js/log";
+import { ByteArray } from "@project-chip/matter.js/util";
+import { BtpCodec } from "@project-chip/matter.js/codec";
+
+const logger = Logger.get('BleBroadcaster');
+
+export class BleBroadcaster implements InstanceBroadcaster {
+    private vendorId: VendorId | undefined;
+    private productId: number | undefined;
+    private discriminator: number | undefined;
+    private advertise = false;
+
+    constructor(
+        private readonly blenoServer: BlenoBleServer,
+        private readonly additionalAdvertisementData?: ByteArray
+    ) { }
+
+    setCommissionMode(mode: number, { deviceName, deviceType, vendorId, productId, discriminator }: CommissioningModeInstanceData) {
+        if (mode !== 1) {
```

Comment:
Could benefit from an enum for mode

Reply:
Formally nothing defined in specs beside 1 and 2 :-) So I would need to invent an enum ... lets stay close to specs ;-)